app/assets/javascripts/welcome.coffee

```diff
@@ -35,3 +35,17 @@ $ ->
       success: (data) ->
         $('#partial_news').html data
     event.preventDefault()
+
+  $('#myCarousel').carousel()
```

Comment:
better move this few initialize rows to some method caled 'initializeCarusel' or some similar. and call it here.